### PR TITLE
[docs] fix markdown link

### PR DESCRIPTION
**Pull request by yamankatby**
# Why

# How

# Test Plan

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Review by Simek — CHANGES_REQUESTED** [review]
Hello @yamankatby, thank you for the update! 👍 

I have left one suggestion, about the second link in sentence, which seems also not to work. Please address that and we can ship the changes. 🙂

File: docs/pages/guides/using-firebase.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ title: Using Firebase
 
 If you'd like to use Firebase in the Expo Go app with the managed workflow, we'd recommend using the [Firebase JS SDK](https://github.com/firebase/firebase-js-sdk). It supports Authentication, Firestore & Realtime databases, Storage, and Functions on React Native. Other modules like Analytics are [not supported through the Firebase JS SDK](https://firebase.google.com/support/guides/environments_js-sdk), but you can use [expo-firebase-analytics](/versions/latest/sdk/firebase-analytics) for that.
 
-If you'd like access to the full suite of native firebase tools, we recommend using the [react-native-firebase](https://github.com/invertase/react-native-firebase) library and creating [a development build] of your app (/development/introduction.md) using a built-in [config plugin].
+If you'd like access to the full suite of native firebase tools, we recommend using the [react-native-firebase](https://github.com/invertase/react-native-firebase) library and creating [a development build](/development/introduction.md) of your app using a built-in [config plugin].
```
[the remark this answers is not in the document]
Comment (yamankatby) [reply docs/pages/guides/using-firebase.md]: done

**Review by Simek — APPROVED** [review]
Thank you for the update, LGTM! 👌

**Comment by yamankatby** [comment]
your quick reply is great. keep it up @Simek

**Pull request merged**